model/project_selector.go

```diff
@@ -172,9 +172,6 @@ func (tse *tagSelectorEvaluator) evalSelector(s Selector) ([]string, error) {
 			results = utility.StringSliceIntersection(results, names)
 		}
 	}
-	if len(results) == 0 {
-		return nil, errors.Errorf("nothing satisfies selector '%v'", s)
```

Comment:
+1

Reply:
Yes! Since this is the evaluation layer (and the latest layer that has access to the raw build variant task selectors- after this layer they are parsed to their tasks), I added a temporary field that won't be saved/sent anywhere just for validation, then I read that in our warnings for build variant function.

After running, I get the warning:
WARNING: buildvariant 'test-this' contains empty task selectors: '.release_critical !.requires_large_host'

But it still passes validation!